### PR TITLE
updated requirements

### PR DESCRIPTION
**Pull request by darrellodonnell**
in process of getting requirements captured and migrated (from v1).

### CODE REVIEW

File: v2/requirements.md [hunk 1]
```diff
@@ -79,16 +80,15 @@ All other terms in **bold** will be defined in one or more ToIP glossaries in th
     - iii. Operational **requirements**.
     - iv. Legal contracts.
 7. MUST specify in its **EGF** (or in any referenced **credential governance framework**) **requirements** for:
-    - i. An **authorized issuer**, including:
-        - a. The **EGF URI** that MUST be included as a **claim** in any authorized **credential**.
-        - b. The **credential type URI** that MUST be used for any authorized **credential**.
-    - ii. An **authorized verifier**, including:
-        a. The **presentation type URI** that an **authorized verifier** MUST use for any authorized **presentation request**.
+    - i. all `authorization` values that are used by the trust registry.
+    - ii. all Assurance Levels, specified with unique names, that are service by the trust registry.
```
Comment (andorsk) [inline v2/requirements.md:84]: What is an Assurance Level?

**Review by andorsk — CHANGES_REQUESTED** [review]
We should add some language around Assurance Levels.

File: v2/requirements.md [hunk 2]
```diff
@@ -68,7 +69,7 @@ All other terms in **bold** will be defined in one or more ToIP glossaries in th
 
 1. MUST have exactly one **primary trust registry**.
 2. MAY have zero or more **secondary trust registries**. (The **primary trust registry** plus all **secondary trust registries** are collectively the **authorized trust registries**.)
```
[the remark this answers is not in the document]
Comment (a-fox) [reply v2/requirements.md:71]: Are we going to link to the glossary for terms, or should we define primary & secondary trust registry here as well?

File: v2/requirements.md (hunk 1 above)
Comment (darrellodonnell) [reply v2/requirements.md:84]: @andorsk updating ToIP glosarry with a definition. The PCTF defines it as "In the Pan-Canadian Trust Framework™ (PCTF), a Level of Assurance (LoA) represents the level of confidence an Entity may place in the processes and other conformance criteria defined in any given component of the PCTF. Levels of Assurance are elemental in creating networks of trust."

@a-fox assurance levels would be defined in the EGF and be ecosystem-specific.

File: v2/requirements.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (darrellodonnell) [reply v2/requirements.md:71]: I think we should refer to the glossary - and thus add any terms that we really need to the ToIP glossary.

**Comment by darrellodonnell** [comment]
need to create issue about Assurance Levels

File: v2/requirements.md (hunk 1 above)
Comment (sankarshanmukhopadhyay) [reply v2/requirements.md:86]: I think the "Assurance Level" concept can be offered as something which can be adopted as is or an equivalent in the jurisdiction is adopted. Reminding the reader that LoA is a critical part of this is necessary.

**Comment by andorsk** [comment]
Discussed at ToIP : Early merge and continue moving on in future PRs. This comments will be addressed in future PRs.

**Pull request merged**